vault/102/README.md

```diff
@@ -0,0 +1,58 @@
+# <a href="http://www.devopsloft.io">DevOps Loft</a>
+
+<img src="http://www.devopsloft.io/static/logo.png" alt="drawing" width="250" hight="250"/>
+
+<details>
+  <summary>Prerequisites</summary>
+  <ul>
+    <li>Git</li>
+    <li>GitHub account</li>
+    <li>Fork this repo</li>
+    <li>Clone this repo</li>
```

Comment:
better write "Clone your fork of this repo"

Reply:
+1